docs/_data/examples.yml

```diff
@@ -0,0 +1,146 @@
+- id: terraform-http-basic
+  name: HTTP Basic
+  image: /assets/img/logos/terraform-logo.png
+  files:
+    - name: main.tf
+      url: /examples/terraform-basic-example/main.tf
+      prism_lang: hcl

```

Comment:
If not specified, could we have a map that picks a reasonable language based on extension? E.g., For `.tf`, default to `hcl`; for `.json`, default to `json`; for `.go`, default to `go`; etc. Again, would help make this file less redundant.